css/styles.scss

```diff
@@ -508,6 +508,17 @@ tr[data-itemtype="PluginFormcreatorCondition"] ~ tr[data-itemtype="PluginFormcre
       border: none;
       padding: 16px;
       height: 51px;
+      span {
+         white-space: nowrap;
+         overflow: hidden;
+         text-overflow: ellipsis;
+         display: inline-block;
+         width: calc(100% - 20px);
+      }
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      width: calc(100% - 20px);
```

Comment:
required for the anchors allowing to go 1 level up in the tree becaise slinky does not wraps them in a SPAN tag.